src/material-experimental/mdc-helpers/_focus-indicators.scss

```diff
@@ -7,19 +7,46 @@
 ///   .my-app {
 ///     @include mat-mdc-strong-focus-indicators($config);
 ///   }
+
+// Mixin that handles null values for CSS variables
+@mixin _create-variable($name, $value) {
+  @if ($value) {
+    --#{$name}: #{$value};
+  }
+}
+
+// Mixin to dedup CSS Variables for the strong-focus-indicators mixin
+@mixin _strong-focus-indicators-helper($config) {
+  $border-style: map.get($config, border-style);
+  $border-width: map.get($config, border-width);
+  $border-radius: map.get($config, border-radius);
+  $border-color: map.get($config, border-color);
+
+  @include _create-variable('mat-focus-indicator-border-style-custom', $border-style);
+  @include _create-variable('mat-focus-indicator-border-width-custom', $border-width);
+  @include _create-variable('mat-focus-indicator-border-radius-custom', $border-radius);
+  @include _create-variable('mat-focus-indicator-border-color-custom', $border-color);
+}
+
 @mixin strong-focus-indicators($config: ()) {
   // Default focus indicator config.
   $default-config: (
     border-style: solid,
```

Comment:
These default focus indicator attributes (i.e. `solid`, `3px`, and `4px`) need to stay in sync with the default values in `_core.scss`. Alternatively, I think we could actually just delete these default attributes and reduce the `$default-config` to just `(border-color: black)` (because then the defaults in `_core.scss` will be applied).

Reply:
Gotcha, sounds great and makes sense! Will go ahead and apply those changes.